aspnetcore/web-api/define-controller.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Define a controller in ASP.NET Core Web API
+author: scottaddie
+description: Learn about the options for defining a controller class in an ASP.NET Core Web API and when it's most appropriate to use each.
+manager: wpickett
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 04/18/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: web-api/define-controller
+---
+# Define a controller in ASP.NET Core Web API
+
+By [Scott Addie](https://github.com/scottaddie)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/web-api/define-controller/samples) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+ASP.NET Core offers the following options for creating a Web API controller:
+
+::: moniker range="<= aspnetcore-2.0"
+* [Derive class from Controller](#derive-class-from-controller)
+* [Derive class from ControllerBase](#derive-class-from-controllerbase)
+::: moniker-end
+::: moniker range=">= aspnetcore-2.1"
+* [Derive class from Controller](#derive-class-from-controller)
+* [Derive class from ControllerBase](#derive-class-from-controllerbase)
+* [Annotate class with ApiControllerAttribute](#annotate-class-with-apicontrollerattribute)
+::: moniker-end
+
+This document explains when it's most appropriate to use each option.
+
+## Derive class from Controller
```

Comment:
The document seems focused on APIs, I think it's good to do a comparison, but explain that `Controller` is recommended for HTML and Razor.

To most users this won't make a big difference, but it's possible in the future we'd ship a remixed framework that doesn't include `Controller` by default.